arabic/unpacked/html.js

```diff
@@ -0,0 +1,90 @@
+MathJax.Hub.Config({
+  'HTML-CSS': {
+    undefinedFamily: 'Amiri',
+    styles: {
+      '.mfliph': {
+        'display': 'inline-block !important',
+        '-moz-transform': 'scaleX(-1)',
+        '-webkit-transform': 'scaleX(-1)',
+        '-o-transform': 'scaleX(-1)',
+        'transform': 'scaleX(-1)',
+        '-ms-filter': 'fliph',
+        'filter': 'fliph'
+      },
+      '.mar': {
+        'font-family': 'Amiri !important',
+        'font-style': 'normal !important'
+      }
+    }
+  }
+});
+
+
+MathJax.Hub.Register.StartupHook('mml Jax Ready', function () {
+  MathJax.Hub.Register.StartupHook('HTML-CSS Jax Ready', function () {
+    MathJax.Hub.Register.StartupHook('Arabic TeX Ready', function () {
+      var MML = MathJax.ElementJax.mml;
+
+      var flipHorizontalElement = function (token, element) {
+        var className = '';
+
+        if ('ar' === token.Get('fontLang')) {
```

Comment:
> The Get() method is used to handle inheritance of properties via <mstyle> and other elements that affect their children. It looks like your fontLang and fliph properties are only used on the elements themselves (not inherited), so there is no need to go through the overhead of Get(). Just use token.fontLang and token.fliph directly.
I was wondering why there's `Get` when I could do simple assignment.
Thanks for the note. Now I know :smile:.

> Also, there is a convention that if you add properties or methods to the base MML classes, that you use a name prefix so help avoid potential conflicts with other extensions. So using something like arabicFontLang and arabicFlipH or something like that would be the normal way to do this. (I haven't been as careful about this as I should have, and a number of extensions don't do this, but we're trying to be more rigorous about it with new code.)
 - [ ] Sure, I like that. Will amend accordingly.